app/models/user.rb

```diff
@@ -35,8 +35,10 @@ def current_streak
     return 'Currently, you don\'t own an active subscription' unless active_subscription
```

Comment:
@castrolem same as above. Also, I don't feel the model should take care of generating messages, we can use a helper or a controller method for this.